test/torchaudio_unittest/datasets/commonvoice_test.py

```diff
@@ -11,55 +12,106 @@
     normalize_wav,
 )
 
+from torchaudio.datasets import COMMONVOICE
 
-class TestCommonVoice(TempDirMixin, TorchaudioTestCase):
-    backend = 'default'
+original_ext_audio = COMMONVOICE._ext_audio
+sample_rate = 48000
```

Comment:
This `sample_rate` variable is shadowed in test function when test method reaches `for i, (waveform, sample_rate, dictionary) in enumerate(dataset):`.

This is an anti-pattern, though the current implementation is fine. 

Can you change it to something like `_SAMPLE_RATE`?
The same goes to other global variables. (Yes, I am aware that I used `original_ext_audio` in my previous example, and sorry about that.)